kalite/distributed/static/js/distributed/topics/router.js

```diff
@@ -41,7 +41,7 @@ ChannelRouter = Backbone.Router.extend({
         var current_url = Backbone.history.getFragment();
         var fragments = current_url.split("/").slice(0,-1);
         if (fragments.length > 0) {
-            this.navigate(fragments.slice(0,-1).join("/") + "/");
+            this.navigate(fragments.slice(0,-1).join("/") + "/", {trigger: true});
```

Comment:
Fortunately, this code is left over from the last time we had a back button in the sidebar (plus ça change), and is not called by any other functions, so should be fine to modify.